Chutzpah/ChutzpahJSRunners/Chrome/chutzpahRunner.js

```diff
@@ -246,12 +248,20 @@ module.exports.runner = async (onInitialized, onPageLoaded, isFrameworkLoaded, o
         }
 
         chutzpahFunctions.rawLog("!!_!! Using Chrome Install : " + chromeExecutable);
-        debugLog("Launch Chrome (" + chromeExecutable +"): Elevated= " + isRunningElevated);
+        debugLog("Launch Chrome (" + chromeExecutable + "): Elevated= " + isRunningElevated);
+
+        // If isRunningElevated, we need to turn off sandbox since it does not work with admin users
+        var launchBrowserArges = isRunningElevated ? ["--no-sandbox"] : [];
+        if (browserArgs) {
+            launchBrowserArges.push(...browserArgs.trim().split(" "));
```

Comment:
options separated by space is consistent with the format as passing on commandline